### PR TITLE
[base] Changed the namespace of Cancellable from my to base.

### PR DESCRIPTION
**Pull request by mpimenov**

### CODE REVIEW

**Comment by mpimenov** [comment]
JTTP

File: base/cancellable.hpp [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
-  /// Marks current activity as not cancelled.
+  // Marks current activity as not cancelled.
   virtual void Reset() { m_cancelled = false; }
 
-  /// Marks current activity as cancelled.
+  // Marks current activity as cancelled.
   virtual void Cancel() { m_cancelled = true; }
 
-  /// \return True is current activity is cancelled.
+  // returns true iff current activity has been cancelled.
```
Comment (tatiana-yan) [inline base/cancellable.hpp]: s/returns/Returns ?

File: search/features_layer_path_finder.hpp [hunk 2]
```diff
@@ -13,7 +13,7 @@
 class FeaturesVector;
 class MwmValue;
 
-namespace my
+namespace base
 {
 class Cancellable;
 }
```
Comment (tatiana-yan) [inline search/features_layer_path_finder.hpp]: `//  namespace base` ?

Comment (mpimenov) [reply search/features_layer_path_finder.hpp]: We don't in one-liners.

File: base/cancellable.hpp (hunk 1 above)
Comment (mpimenov) [reply base/cancellable.hpp]: done

**Pull request merged**